docs/parcels.md

```diff
@@ -0,0 +1,110 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be controlled imperatively 
+by an application, parcels use similar methodology as applications, and can be considered an advanced single spa feature so we recommend [starting with applications.](/docs/applications.md)
```

Comment:
I'm curious to hear your thoughts on this -- why is parcels an advanced feature? Why do you recommend you start with applications? What if someone just wants to use parcels as a framework agnostic component without using single-spa applications?

I actually think that parcels might be useful in a lot of situations and that they aren't scary and advanced.

Reply:
I don't have really strong feeling about it, I've been considering them advanced because they take more configuration than an application.

Reply:
What are the situations?